project.yaml

```diff
@@ -61,30 +102,33 @@ actions:
       moderately_sensitive:
         measure_csv: output/data/measure_*.csv
 
-
   ## Process data
-  #data_process:
-  #  run: r:latest analysis/process/process_data.R
-  #  needs: [join_ethnicity, generate_measures]
-  #  outputs:
-  #    moderately_sensitive:
-  #      measure_csv: output/data/measure_final_*.csv
+  process_data:
+   run: r:latest analysis/process/process_data.R
+   needs: [join_cohorts, generate_measures]
```

Comment:
You don't need `join_cohorts` here. It is already included in  `needs` for `generate_measures`.

Reply:
Actually I think I do need it here (doesn't run otherwise) b/c I need to access person-level data for specific months to create the file needed for the demographics table.